core/callable_method_pointer.h

```diff
@@ -131,7 +131,7 @@ void call_with_variant_args_helper(T *p_instance, void (T::*p_method)(P...), con
 #ifdef DEBUG_METHODS_ENABLED
 	(p_instance->*p_method)(VariantCasterAndValidate<P>::cast(p_args, Is, r_error)...);
 #else
-	(p_instance->*p_method)(VariantCaster<P>::cast(p_args[Is])...);
+	(p_instance->*p_method)(VariantCaster<P>::cast(*p_args[Is])...);
```

Comment:
Now it does the same thing that happens in `VariantCasterAndValidate<P>::cast` after checks:
```
VariantCaster<T>::cast(*p_args[p_arg_idx]);
```